playwright-test-js/tests/test_2.spec.js

```diff
@@ -4,6 +4,8 @@ const { expect } = require('@playwright/test')
 test.describe('Browse LambdaTest in different search engines', () => {
   test('Search LambdaTest Blog on Bing', async ({ page }) => {
     await page.goto('https://www.bing.com')
+    await page.waitForLoadState('domcontentloaded')
```

Comment:
Is it failing without the waits?

Reply:
@ankur-lt it fails Intermittently. Gave the reason in the above comment. Pls check